src/js/call.js

```diff
@@ -127,6 +127,14 @@ Call.isIpv6 = function(candidate) {
   return candidate.address.indexOf(':') !== -1;
 };
 
+Call.isUdp = function(candidate) {
+  return candidate.protocol === 'udp';
```

Comment:
Actually, we need to check for reflexive candidates here in order to verify that you have reached a STUN server, it's type `srflx` which actually stand for server reflexive candidates (provided by the stun server).